stdlib/public/core/StringStorage.swift

```diff
@@ -769,35 +822,51 @@ extension __SharedStringStorage {
 
 // Get and populate breadcrumbs
 extension _StringGuts {
+  
+  @inline(never)
+  @_effects(releasenone)
+  internal func createAndLoadBreadcrumbs_time_here_is_String_to_NSString_bridging_overhead(
```

Comment:
Pulling this out into a separate function is partly so I can give it a descriptive name for people doing profiling, but also so that we can force-inline the fast paths.